odoo/modules/loading.py

```diff
@@ -431,3 +431,25 @@ def load_modules(db, force_demo=False, status=None, update_module=False):
             _logger.log(25, "All post-tested in %.2fs, %s queries", time.time() - t0, odoo.sql_db.sql_counter - t0_sql)
     finally:
         cr.close()
+
+
+def reset_modules_state(db):
+    """
+    Resets modules flagged as "to x" to their original state
+    """
+    # Warning, this function was introduced in response to commit 763d714
+    # which locks cron jobs for dbs which have modules marked as 'to %'.
+    # The goal of this function is to be called ONLY when module
+    # installation/upgrade/uninstallation fails, which is the only known case
+    # for which modules can stay marked as 'to %' for an indefinite amount
+    # of time
+    with db.cursor() as cr:
+        cr.execute(
+            "UPDATE ir_module_module SET state='installed' WHERE state IN "
```

Comment:
Is it possible to keep query on 1 line?
max-line-length 99 is OK.

Reply:
I am a PEP8 purist :)

Reply:
pep8 tolerate a 99 limit, which is fine ;)

Reply:
I'll change it if you *really really* think it'll improve readibility, but to me > 79 is horrible